### DISCUSSION TITLE
Timeline layout more fit for single-frame workflow

### DISCUSSION THREAD

**Discussion by novhack** [Orama-Interactive/Pixelorama#467] [Ideas]
The current Pixelorama layout is heavily leaning towards animation with focus on horizontal space for frames.
I propose a toggle-able mode that would move the timeline to the right side and use more vertical space.

I can imagine using Pixelorama in the "single-frame" mode even for animation to first create a static image with lot of layers and then switch to "animation" mode to make it move.

This is a very quick shuffle of nodes that I did to illustrate how it could look (this is already fully usable but not pretty):
![image](https://user-images.githubusercontent.com/42614907/111954472-71218500-8ae8-11eb-8b21-38d03dbaef20.png)

**Comment by yuyutsupoudel** [comment]
Looks really nice but more compact ui would be better. Nearly half of screen is covered by tool related UI and only half of screen is actual working place.

**Comment by OverloadedOrama** [comment]
This should now be possible with the new UI system.

**Comment by dphaldes** [comment]
I would also suggest to split or duplicate the animation timeline's layer system into 2 separate dialogs. One containing only layers on a specific frame and the second one containing layers + all frames. The user will be able to enable these panels independently and thus create their own workflow.
